standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Text.Json.xml

```diff
@@ -213,14 +217,14 @@
       <summary>Gets the value of the element as a <see cref="T:System.DateTime" />.</summary>
       <returns>The value of the element as a <see cref="T:System.DateTime" />.</returns>
       <exception cref="T:System.InvalidOperationException">This value's <see cref="P:System.Text.Json.JsonElement.ValueKind" /> is not <see cref="F:System.Text.Json.JsonValueKind.String" />.</exception>
-      <exception cref="T:System.FormatException">The value cannot be represented as a <see cref="T:System.DateTime" />.</exception>
+      <exception cref="T:System.FormatException">The value cannot be read as a <see cref="T:System.DateTime" />.</exception>
```

Comment:
@ahsonkhan you can confirm your changes are here 😃

Reply:
Yep, they are here :)